spec/features/trainer_spec.rb

```diff
@@ -223,6 +231,7 @@
     before do
       create(:user_with_two_blocks_without_cards, current_block_id: 1)
       visit trainer_path
+      click_link 'ru'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.